### ISSUE TITLE
How to set margin top on all pages?

### ISSUE THREAD

**Issue by golubvladimir** [marcbachmann/node-html-pdf#675]
I created project - https://github.com/golubvladimir/nestjs-generate-pdf

I want to create document with same header on all pages. I created header by element with id pageHeader.
After I set height to 1px in options.
```
header: {
	height: '1px'
}
```
I don't know, how to set margin for all pages from header. Now it works only on first page.

**Comment by golubvladimir** [comment]
I have this problem only on Windows.

**Comment by matthew-boston** [comment]
I don't think you can set margins explicitly. However, if you apply a margin to the body like so:
```
body{
  margin: 30px;
}
```
This will apply a 30px margin around the content inside the document.

**Comment by DaraSingh1998** [comment]
It only adds the top margin on 1st page. How to do this at all page
